.gitignore

```diff
@@ -1,29 +1,44 @@
-*.swp
-.DS_Store
-.bundle
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
```

Comment:
This is a fine/positive change, but when the time comes we should ensure this gets added in to onboarding/early dev setup instructions.  I fear a future where we start seeing accidental commits of DS_Store files because this change occurred somewhat silently and obscured by the larger Rails upgrade.

Reply:
I'm not too worried about that, as this is the only one of our repos that has ignored the `.DS_Store` files. Various devs have occasionally committed various types of swap files (not just DS_store), and we've just addressed it at that point.